src/main/java/bio/terra/cli/command/resources/resolve/BqDataset.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.cli.command.resources.resolve;
+
+import static bio.terra.cli.service.WorkspaceManager.getBigQueryDatasetPath;
+
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.command.helperclasses.FormatOption;
+import bio.terra.cli.service.WorkspaceManager;
+import bio.terra.workspace.model.GcpBigQueryDatasetAttributes;
+import bio.terra.workspace.model.ResourceDescription;
+import picocli.CommandLine;
+
+/** This class corresponds to the fourth-level "terra resources resolve bq-dataset" command. */
+@CommandLine.Command(
+    name = "bq-dataset",
+    description = "Resolve a Big Query dataset resource to its cloud id or path.")
+public class BqDataset extends BaseCommand {
+  @CommandLine.Option(
+      names = "--name",
+      required = true,
+      description = "Name of the resource, scoped to the workspace.")
+  private String name;
+
+  @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+  BqDataset.BqDatasetCloudIdArgGroup argGroup;
+
+  static class BqDatasetCloudIdArgGroup {
+    @CommandLine.Option(
+        names = "--dataset-id",
+        description = "Only return the Big Query dataset id.")
+    private boolean datasetId;
+
+    @CommandLine.Option(names = "--project-id", description = "Only return the GCP project id.")
```

Comment:
is there a way to describe what's returned if neither is set? Does an enum make more sense for this tri-state, or is there a reason to prefer using exclusive flags? Naively, an enum parameter with a default value seems more self documenting.

Reply:
Changed to use an enum.

Reply:
No, It doesn't appear by default. I'll try making it an enum instead and see what you think.

Reply:
Changed from exclusive flags -> enum.
<img width="909" alt="Screen Shot 2021-04-29 at 4 48 23 PM" src="https://user-images.githubusercontent.com/12446128/116616499-c6805b80-a90a-11eb-8705-7e6643696839.png">